akka-persistence/src/main/scala/akka/persistence/PersistencePlugin.scala

```diff
@@ -80,7 +80,12 @@ private[akka] abstract class PersistencePlugin[ScalaDsl, JavaDsl, T: ClassTag](s
 
     instantiate(
       (classOf[ExtendedActorSystem], system) :: (classOf[Config], pluginConfig) ::
-      (classOf[String], configPath) :: Nil)
+      (classOf[String], configPath) :: (classOf[Config], mergedConfig) :: Nil)
+      .recoverWith {
+        case _: NoSuchMethodException =>
+          instantiate((classOf[ExtendedActorSystem], system) :: (classOf[Config], pluginConfig) ::
+          (classOf[String], configPath) :: Nil)
+      }
```

Comment:
Not convinced that the special 2-config signature should be the first one to try though, most plugins will probably not implement it. Better push it further down the recover list IMO

Reply:
Agree. It's been done.